src/js/forms.js

```diff
@@ -134,6 +135,59 @@ class Forms {
 		this.FormEl = undefined;
 	}
 
+	static updateInputStatus (input, status) {
+		if (!(input instanceof HTMLElement)) {
+			throw new TypeError(`Expecting "input" to be an instance of "HTMLElement" but instead found ${oFormsEl instanceof HTMLElement}.`);
```

Comment:
I don't _fully_ understand this error. So will it output the following?

```
Expecting "input" to be an instance of "HTMLElement" but instead found true.
```

Reply:
Hah that is a mistake. I'll update `${oFormsEl instanceof HTMLElement}` to `${typeof oFormsEl}`